docs/user/overview.md

```diff
@@ -2,21 +2,100 @@
-The repository is also responsible for importing and  exporting CSARs, the exchange format of TOSCA files and related artifacts.
+In addition to the described basis functionality of the TOSCA modeling tool Winery several advanced functionalities are provided:
+
+- *Consistency Check*: This functionality enables to check whether a service template is valid according to the TOSCA
+XML specification. This includes the definition of used node types and properties, the QNames, and if License and README files are available.
+This supports the user to model valid service templates. 
+
+- *XaaS Packager*: It enables the deployment of an, e.g., web application by reusing an existing service template and 
+replacing the deployment artifact in the specified node type with the new deployment artifact. The underlying
+platform or infrastructure services do not have to modeled for each application, predefined templates can be used. 
+More information can be found [here](XaaSPackager).
+
+- *Topology Completion*: The TOSCA Topology Completion of Winery enables the user to model incomplete 
+TOSCA Topology Templates and complete them automatically step-by-step by injecting new node templates to fulfill open requirements.
+More information can be found [here](TopologyCompletion).
+
+- *Splitting & Matching*: The Split & Match function facilitates the redistribution of application components to target locations.
+For this, the application components can be annotated with target labels to indicate the desired 
+target locations.
+In the *Matching Templates Repository* platform or infrastructure services can be defined as node templates or complete topology fragements
+for each target location. Based on the desired split, the node templates of the original service template are split according
+to the labels and the matching node templates or topology fragments for hosting the application's components in the
+target location are matched with the corresponding part of the split topology.
+More information can be found [here](Splitting).
+
+- *Versioning & Difference Calculation*: To support version control of all TOSCA elements, including node types, artifact
+templates, service templates, and so on, the versioning component enables to add different versions of a TOSCA element
+and to release them after the development phase. Released elements can not be modified to ensure consistency of specific
+versions in the ecosystem. In addition, the differences between two versions can be calculated and visualized in the
+TOSCA Topology Model Editor.
+
+- *Accountability*: In collaborative development of application deployment models in business-critical scenarios accountability
+is of high importance. Thus, the Winery enables to store the TOSCA meta file in a blockchain to identify the author of each version
+of a CSAR and whether an artifact is changed and by whom. This facilitates to visualize the provenance of a specific
+resource.
```

Comment:
*Accountability*: In collaborative development of application deployment models in business-critical scenarios (such as data-analysis), accountability is of high importance. Thus, at CSAR export time, Winery enables to store the TOSCA meta file in a blockchain to identify the author of each exported version and whether a contained artifact is changed and by whom. 
Winery also stores these artifacts versions in a decentralized storage which facilitates comparing them and visualizing the provenance of a specific resource.